src/components/Editor/Editor.tsx

```diff
@@ -17,13 +19,20 @@ type EditorComponent = React.ForwardRefExoticComponent<
 
 const Editor: EditorComponent = forwardRef(
   ({ type, code, fontSize, extensions, onChange }, ref) => {
+    const { theme: storybookTheme } = useStorybookState();
+    const { editorTheme: addonTheme } = useParameter<PlaygroundParameters>(
+      ADDON_ID_FOR_PARAMETERS,
+      DEFAULT_ADDON_PARAMETERS
+    );
     const placeholder = `Insert your ${type.toUpperCase()} code here`;
+    const theme: EditorColor =
+      addonTheme || storybookTheme.base === "dark" ? "dark" : "light";
```

Comment:
```suggestion
    const theme: EditorColor =
      addonTheme || storybookTheme.base;
```

Reply:
It can also be "normal", that's the reaosn for it 😥

Reply:
<img src="https://media4.giphy.com/media/I4UhtUMzOrQSQ/giphy.gif"/>